test/integration/inventory

```diff
@@ -6,6 +6,7 @@ testhost3 ansible_ssh_host=127.0.0.3
 testhost4 ansible_ssh_host=127.0.0.4
 # For testing fact gathering
 facthost[0:25] ansible_host=127.0.0.1 ansible_connection=local
+aws_local
```

Comment:
Files specific to a test target should be under that target's directory tree, not directly under `test/integration/`.